ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -770,4 +770,7 @@ export default defineComponent({
   height: 35px !important;
   width: 35px;
 }
+::v-deep .registration-type-select .v-select__selections:first-child {
+  width: 125px;
```

Comment:
Is this actually the width being applied?
Seems much wider from the screen shots!

Reply:
This is only the width of the inner input.

![image](https://github.com/bcgov/ppr/assets/77707952/7548da4f-b173-4de7-93ce-3844df0c8398)

It matches of the autocomplete component in PPR. Although there is minor styling difference to the way the text is displayed because of difference in components, and I don't believe trying to fix that is a good use of time.

![image](https://github.com/bcgov/ppr/assets/77707952/a0ae986b-0ab9-4ff9-b9a9-935b10379cc7)

Reply:
Indeed!